messages/pt.json

```diff
@@ -1,7 +1,15 @@
 {
-  "checkout-payment.button.save": "Salvar novo cartão",
-  "checkout-payment.input.document": "Documento do titular do cartão",
-  "checkout-payment.input.requiredField": "Campo obrigatório.",
-  "checkout-payment.input.invalidDigits": "Dígitos inválidos",
-  "checkout-payment.credit-card.label": "Cartão de Credito"
+  "store/checkout-payment.installmentsButton": "Escolher parcelamento",
+  "store/checkout-payment.documentLabel": "Documento do titular do cartão",
+  "store/checkout-payment.requiredField": "Campo obrigatório.",
+  "store/checkout-payment.invalidDigits": "Dígitos inválidos",
+  "store/checkout-payment.creditCardLabel": "Cartão de Crédito",
+  "store/checkout-payment.paymentOptionLabel": "Escolha uma opção de pagamento",
+  "store/checkout-payment.installmentValue": "{installments}x {value}",
+  "store/checkout-payment.interestFree": "Sem juros",
+  "store/checkout-payment.newCreditCardLabel": "Novo Cartão de Crédito",
+  "store/checkout-payment.incompleteCardLabel": "Cartão Incompleto",
```

Comment:
Why the upper case for the `I` letter in `Incompleto`?

Reply:
I don't know, this feature was implemented without design, but the other messages are cased consistently with this message.